run_battle_test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+IFS=$'\n'
+
+DIRECTORY=units
+
+for file1 in `ls $DIRECTORY`
+do
+    for file2 in `ls $DIRECTORY`
+    do
+        if [[ $file1 != $file2 ]]
+        then
+            ./$1 $DIRECTORY/$file1 $DIRECTORY/$file2 >> $2
+        fi
+    done
+done
```

Comment:
Ez szep altalanos, de ugye epul arra a feltetelezesre, hogy biztosan ugyanolyan sorrendben megy majd mindig vegig a for  a fajlokon.  Nem nagy gaz, csak mondom.

Reply:
Ezt is át kell írni ahhoz hogy elfogadd a pull requestet, vagy elég ha csak *feltételezzük* hogy jól működik? 😁 
Vagy az `ls`-t hívjam meg úgy hogy rendezze le pl.: abc sorrendbe vagy fájl létrehozása szerint és akkor ugyan az lesz a sorrend?

Reply:
Maradhat csak jeleztem, hogy "instabil" Ha legalabb fixalva lenne, miylen sorrendben vannak listazva a file-ok, akkor mar egy fokkal jobb.